### PR TITLE
Lint and refactor

### PR DESCRIPTION
**Pull request by liyasthomas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,8 +69,8 @@ _Customized themes are synced with local session storage_
  - Instant loading with [Service Workers](https://developers.google.com/web/fundamentals/primers/service-workers)
  - Offline support
  - Low RAM/memory and CPU usage
- - [Add to Home Screen](https://developers.google.com/web/fundamentals/app-install-banners) (button in footer)
- - [Desktop PWA](https://developers.google.com/web/progressive-web-apps/desktop) support (button in footer)
```
Comment (jamesgeorge007) [inline README.md]: Why did you remove those anchors?

Comment (liyasthomas) [reply README.md]: Links have been moved from footer to header which is pretty visible. That's why.
